<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stashadd</class>
 <widget class="QWidget" name="stashadd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1944</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仓库出入单添加界面</string>
  </property>
  <widget class="QLabel" name="IDLable">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>入库单号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="StashLable">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>36</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>入库仓库:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="selectGoodTableWidget">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>120</y>
     <width>821</width>
     <height>691</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>货品编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>货品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单位</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>采购单位价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>统计金额</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>820</y>
     <width>210</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="doneButton">
      <property name="text">
       <string>完成</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="selectstashCombox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>46</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>9</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>系统自动生成</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="availableTableWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>118</y>
     <width>1091</width>
     <height>691</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>货品编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>货品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单位</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>库存量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>参考价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>供应商公司</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>84</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>可选择商品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>926</x>
     <y>84</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>已选择商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>84</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAllButton">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>813</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总价格：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalPriceEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1158</x>
     <y>813</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>自动统计</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>57</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>57</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>84</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>单位价格</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEdit">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>87</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>默认参考价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>840</y>
     <width>497</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>制单人：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string notr="true">不可修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>制单时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>不可修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
